docs/Cask-Cookbook.md

```diff
@@ -502,17 +502,17 @@ The available symbols for hardware are:
 | ---------- | -------------- |
 | `:x86_64`  | 64-bit Intel   |
 | `:intel`   | 64-bit Intel   |
+| `:arm64`   | Apple M1   |
```

Comment:
M1 is the specific processor model (there will probably be an M2 in the future), I'd suggest this instead:

```suggestion
| `:arm64`   | Apple Silicon |
```